src/Security/UserClassBuilder.php

```diff
@@ -232,7 +232,7 @@ private function addGetSalt(ClassSourceManipulator $manipulator, UserClassConfig
         if ($userClassConfig->hasPassword()) {
             $builder->addStmt(
                 $manipulator->createMethodLevelCommentNode(
-                    'not needed when using the "bcrypt" algorithm in security.yaml'
+                    'not needed when using the "bcrypt/sodium" algorithm in security.yaml'
```

Comment:
Also not needed for `auto` or `argon2i`. Maybe we should just say:

> not needed because we are using a modern encoder algorithm in `security.yaml`

Reply:
> not needed when using a modern encoder algorithm in security.yaml ?

Maybe? _Because_ presume that a modern encoder is used, but it's not necessarly the case right?